src/DotNetProjectFile.Analyzers/Analyzers/MsBuild/RunNuGetSecurityAuditsOnAll.cs

```diff
@@ -0,0 +1,20 @@
+namespace DotNetProjectFile.Analyzers.MsBuild;
+
+/// <summary>Implements <see cref="Rule.RunNuGetSecurityAuditsOnAll"/>.</summary>
+[DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+public sealed class RunNuGetSecurityAuditsOnAll() : MsBuildProjectFileAnalyzer(Rule.RunNuGetSecurityAuditsOnAll)
+{
+    /// <inheritdoc />
+    public override IReadOnlyCollection<ProjectFileType> ApplicableTo => ProjectFileTypes.ProjectFile;
+
+    /// <inheritdoc />
+    protected override void Register(ProjectFileAnalysisContext<MsBuildProject> context)
+    {
+        var property = context.File.Property<NuGetAuditMode>();
```

Comment:
Starting with .NET 10, `all` will be the default. If multi-targeting and it includes the .NET 10 target, it will be `all` for all targets. source: https://learn.microsoft.com/en-us/nuget/concepts/auditing-packages

Should we take that in consideration in the reporting?

Reply:
That is the same with `NuGetAudit` which is enabled since .NET 8. We then decided that we would not take that into account, but we could change it. That would be a seperate PR anyway I would argue. What do you think?

Reply:
Sounds good to me 👍